client/src/components/WorkspacePanels/BoxStyle.css

```diff
@@ -21,7 +21,7 @@
   display: flex;
   justify-content: center;
   align-content: center;
-  flex-direction: column;
+  flex-direction: row;
```

Comment:
Isn't `flex-direction` itself set to `row` by default? Or are we like overriding the value?